tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -793,8 +793,11 @@ internal static class FeatureFlags
             /// <summary>
             /// Enables injecting 128-bit trace ids into logs as a hexadecimal string.
             /// If disabled, 128-bit trace ids will be truncated to the lower 64 bits,
-            /// and all trace ids will be injected as decimal strings
-            /// Default value is <c>false</c> (disabled).
+            /// and injected as decimal strings. 64-bit trace ids are
+            /// always injected as decimal strings, regardless of this setting.
+            /// Default value is <c>true</c> (enabled).
+            /// If <see cref="ConfigurationKeys.FeatureFlags.TraceId128BitGenerationEnabled"/> is set to <c>false</c>,
+            /// this setting will also be set to <c>false</c>.
```

Comment:
Is this the correct behavior? Even if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=false`, we can always receive 128-bit trace ids from upstream and inject them into logs.

Reply:
To answer your question, yes `DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED` can be enabled even if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED` is disabled. We're taking the stance that if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=false` then `DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED` will also be set to `false`, unless the user overrides this with configuration.

I think maybe the documentation doesn't reflect that accurately enough that the user can override this. Would that help?

Reply:
This is being addressed in #6791